tests/test_market.py

```diff
@@ -132,6 +133,59 @@ def market_matcher():
     return PayAsBidMatcher()
 
 
+class TestTwoSidedMarket:
+    """Class Responsible for testing two sided market's functionality.
+
+    TODO: bring all related two sided market tests from this module into this class.
```

Comment:
I think it is even time to create a separate module for this class even now, right?

Reply:
That would be good in my opinion too

Reply:
I agree, but grouping all two sided market tests from this module can take long time and this is PR cannot wait as the stakeholders (research team) is regularly asking for updates and deadlines
should I create a new module and leave the TODO comment in this module ?

Reply:
IMO yes, you can proceed like this.